### PR TITLE
licence + codefactor badges

### PR DESCRIPTION
**Pull request by kelson42**

### CODE REVIEW

**Comment by kelson42** [comment]
@Jaifroid I wanted to add a badge for the CI on `main` too... but was not clear to me "what is actually the CI workflow?".

**Comment by Jaifroid** [comment]
@kelson42 Currently the CI workflow checks that a PR/push can successfully build the WASM versions. At the moment, it does this from scratch. When I've completed a PR on #12, it will do exactly the same, but by pulling `libzim.a` and dependencies from the appropriate openzim/release archive. This is a good check for this repo, I think. In an ideal world, we would add some testing, by automating the demo version, that the WASM is good for reading an archive and doing a search. But for now, it's probably a good enough test that the WASM and glue files can all be built successfully agains the PR. Does that sound right?

**Comment by kelson42** [comment]
> @kelson42 Currently the CI workflow checks that a PR/push can successfully build the WASM versions. At the moment, it does this from scratch. When I've completed a PR on #12, it will do exactly the same, but by pulling `libzim.a` and dependencies from the appropriate openzim/release archive. This is a good check for this repo, I think. In an ideal world, we would add some testing, by automating the demo version, that the WASM is good for reading an archive and doing a search. But for now, it's probably a good enough test that the WASM and glue files can all be built successfully agains the PR. Does that sound right?

Sounds perfect. Current CI is heavy because of the architectural choice. Once you won't compile all the dependencies anymore then we could run it on `main` at each push.... and put appropriate badge in `README.md`. That badge is important, at least to me, to quickly now if OK with current code base (and can be released for example).

File: README.md [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
-There is also an HTML/JS utility for testing the ability of Emscripten File Systems to read large files (muliti-gigabyte) at https://openzim.github.io/javascript-libzim/tests/test_large_file_access/.
+This Repository provides the source code and utilities for compiling
+the [ZIM File](https://wiki.openzim.org/wiki/ZIM_file_format) reader
+[lbizim](https://wiki.openzim.org/wiki/Libzim) from C++ to
+[WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly)
+(and
+[ASM.js](https://developer.mozilla.org/en-US/docs/Games/Tools/asm.js)).
+
+A prototype in HTML/JS, for testing the WASM version, is provided at
+https://openzim.github.io/javascript-libzim/tests/prototyep/. This
```
[the remark this answers is not in the document]
Comment (kelson42) [reply README.md:11]: OK but https://openzim.github.io/javascript-libzim/tests/prototype/ gives 404!

[the remark this answers is not in the document]
Comment (Jaifroid) [reply README.md:11]: Hmm, looks like ghpages isn't updating. Maybe I should do this as a separate PR, then and include the ghpages bot.

**Pull request merged**